src/pages/cart/main/Card.jsx

```diff
@@ -0,0 +1,61 @@
+import { useCart } from "contexts/CartContext";
+import { useWishlist } from "contexts/WishlistContext";
+
+export const Card = (cart) => {
+  const { dispatch } = useWishlist();
+  const { cartState, cartDispatch } = useCart();
+
+  return (
+    <main className="sub-container">
+      {cartState.cart.map((cart) => (
```

Comment:
suggestion : cart.map would have your items or products so instead of using .map(cart) u can also go for item, product.

Reply:
got it